rest-client/src/main/java/org/jboss/pnc/restclient/websocket/VertxWebSocketClient.java

```diff
@@ -49,23 +67,68 @@
 
     private static final Logger log = LoggerFactory.getLogger(VertxWebSocketClient.class);
 
-    private static final ObjectMapper objectMapper = JsonOutputConverterMapper.getMapper();
+    private static final ObjectMapper objectMapper = getObjectMapper();
+
+    /**
+     * Almost identical version of {@link JsonOutputConverterMapper} but without constant error messages on missing
+     * openshift class
+     * 
+     * @return JSON mapper
+     */
+    private static ObjectMapper getObjectMapper() {
+        return new ObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_NULL)
```

Comment:
Shared ObjectMapper should be used for easier maintenance.

Reply:
@matejonnet I've had to use a dedicated ObjectMapper because of NCL-6308. Changes to shared ObjectMapper made WSClient throw errors on every use.